documentation/index.rst

```diff
@@ -13,7 +13,7 @@ Python 3.  It is intended to support codebases that work on both Python 2 and 3
 without modification.  six consists of only one Python file, so it is painless
 to copy into a project.
 
-Six can be downloaded on `PyPi <https://pypi.python.org/pypi/six/>`_.  Its bug
+Six can be downloaded on `PyPI <https://pypi.org/project/six/>`_.  Its bug
```

Comment:
Warehouse still has a giant banner about how it's preproduction, so let's not link there quite yet.

Reply:
Ok. I reverted the URL. Thanks.